modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AbstractScalaLagomCodegen.java

```diff
@@ -0,0 +1,190 @@
+package io.swagger.codegen.languages;
+
+import io.swagger.codegen.CliOption;
+import io.swagger.codegen.CodegenConstants;
+import io.swagger.codegen.CodegenOperation;
+import io.swagger.codegen.DefaultCodegen;
+import io.swagger.models.properties.*;
+
+import java.io.File;
+import java.util.*;
+
+public abstract class AbstractScalaLagomCodegen extends DefaultCodegen {
```

Comment:
@gmkumar2005 Instead of creating another abstract class, what about using [AbstractScalaCodegen.java](https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AbstractScalaCodegen.java) instead?

Reply:
Done